bot/code_review_bot/tasks/coverity.py

```diff
@@ -54,7 +55,7 @@ def __init__(self, analyzer, revision, issue, file_path):
             line=issue["line"],
             nb_lines=1,
             check=issue["flag"],
-            level="error",
+            level=Level.Error,
```

Comment:
Coverity should be warnings, at least for now

Reply:
Done.